deb/build

```diff
@@ -40,4 +40,22 @@ fpm -s dir -t deb -n "$pkgname" -v "$pkgversion" \
 	--depends libtool \
 	--deb-changelog "$changelog" \
 	--deb-no-default-config-files \
-	install/=/
+	install/usr/lib/=/usr/lib/
+
+pkgname=libdrizzle-redux-dev
```

Comment:
AFAIU from Lucas example, the dev version specifies normal version as dependency and only installs headers additionally,